nss-evening-cohort-13/pants-group-project-pantaloons-team-1
# Joe FAQ

## Description
I added a FAQ page with a search function that looks at the FAQ's printed to the DOM

## Related Issue
Closes #35
Closes #34  

## Motivation and Context
Needed a FAQ page that a user could look to for answer to their questions.
Needed a search feature that a user could use to filter through the FAQ's

## How Can This Be Tested?
```# Check to make sure that your working tree is clean. Add and commit all that you need to.
git checkout master
git fetch
git branch -al
# ^ The code above shows you all active branches that are available to you on GitHub.
git checkout your-teammates-branch-name
# Test test test
```

## Screenshots (if appropriate):

## Types of changes
